.evergreen/run-encryption-tests-local.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+tox -e aws-secrets -- --profile "$1" drivers/csfle
+
+source ./secrets-export.sh
+
+export AWS_ACCESS_KEY_ID=$FLE_AWS_KEY
+export AWS_SECRET_ACCESS_KEY=$FLE_AWS_SECRET
+export AWS_DEFAULT_REGION=us-east-1
+export AWS_SESSION_TOKEN=
+
+tox -e encryption-tests
+
+rm -rf libmongocrypt*
+# shellcheck disable=SC2046
+kill -9 $(pgrep -f 'python')
```

Comment:
We can't kill all python processes since that would disrupt other things running on the system.

Reply:
Whoops. this script was for testing, included it by mistake.